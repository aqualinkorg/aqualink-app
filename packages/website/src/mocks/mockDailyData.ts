import { DailyData } from "../store/Reefs/types";

const now = new Date();
const minutesAgo = 5;
const dailyDataDate = new Date(
  now.getTime() - minutesAgo * 60000
).toISOString();

export const mockDailyData: DailyData = {
  id: 1,
  date: dailyDataDate,
  minBottomTemperature: 25,
  maxBottomTemperature: 27,
  avgBottomTemperature: 26,
  degreeHeatingDays: 29,
  surfaceTemperature: 36,
  satelliteTemperature: 20,
  minWaveHeight: 1,
  maxWaveHeight: 3,
  avgWaveHeight: 2,
  waveDirection: 136,
  wavePeriod: 15,
  minWindSpeed: 3,
  maxWindSpeed: 5,
  avgWindSpeed: 4,
  windDirection: 96,
  weeklyAlertLevel: 3,
};
