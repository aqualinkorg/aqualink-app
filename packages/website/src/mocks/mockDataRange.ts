import { TimeSeriesDataRange } from 'store/Sites/types';

export const mockDataRange: TimeSeriesDataRange = {
  bottomTemperature: [
    {
      type: 'hobo',
      depth: null,
      data: [
        {
          minDate: '2017-11-02T15:00:00.000Z',
          maxDate: '2020-02-02T21:00:00.000Z',
        },
      ],
    },
  ],
  cholorophyllConcentration: [
    {
      type: 'sonde',
      depth: null,
      data: [
        {
          minDate: '2017-11-02T15:00:00.000Z',
          maxDate: '2020-02-02T21:00:00.000Z',
        },
      ],
    },
  ],
  odoConcentration: [
    {
      type: 'sonde',
      depth: null,
      data: [
        {
          minDate: '2017-11-02T15:00:00.000Z',
          maxDate: '2020-02-02T21:00:00.000Z',
        },
      ],
    },
  ],
  ph: [
    {
      type: 'sonde',
      depth: null,
      data: [
        {
          minDate: '2017-11-02T15:00:00.000Z',
          maxDate: '2020-02-02T21:00:00.000Z',
        },
      ],
    },
  ],
  salinity: [
    {
      type: 'sonde',
      depth: null,
      data: [
        {
          minDate: '2017-11-02T15:00:00.000Z',
          maxDate: '2020-02-02T21:00:00.000Z',
        },
      ],
    },
    {
      type: 'hui',
      depth: null,
      data: [
        {
          minDate: '2022-06-11T00:00:00.000Z',
          maxDate: '2022-06-14T23:45:00.000Z',
        },
      ],
    },
  ],
  turbidity: [
    {
      type: 'sonde',
      depth: null,
      data: [
        {
          minDate: '2017-11-02T15:00:00.000Z',
          maxDate: '2020-02-02T21:00:00.000Z',
        },
      ],
    },
  ],
};
