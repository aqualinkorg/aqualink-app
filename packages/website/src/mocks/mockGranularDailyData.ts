import { DailyData } from 'store/Sites/types';

export const mockGranularDailyData: DailyData[] = [
  {
    id: 3,
    date: '2022-06-27T02:59:59.999Z',
    minBottomTemperature: 0,
    maxBottomTemperature: 0,
    avgBottomTemperature: 0,
    degreeHeatingDays: 0,
    topTemperature: 0,
    satelliteTemperature: 24.219252436179815,
    minWaveHeight: 0.17217956015502844,
    maxWaveHeight: 0.4220004534334179,
    avgWaveHeight: 0.28517965469411594,
    waveMeanDirection: 219,
    waveMeanPeriod: 2,
    minWindSpeed: 1.0233644143488965,
    maxWindSpeed: 5.558413659400355,
    avgWindSpeed: 2.9796803902255204,
    windDirection: 218,
    weeklyAlertLevel: 0,
  },
  {
    id: 2,
    date: '2022-06-26T02:59:59.999Z',
    minBottomTemperature: 0,
    maxBottomTemperature: 0,
    avgBottomTemperature: 0,
    degreeHeatingDays: 0,
    topTemperature: 0,
    satelliteTemperature: 24.229605847456323,
    minWaveHeight: 0.06706502244580449,
    maxWaveHeight: 0.40127747690701104,
    avgWaveHeight: 0.20286431769935753,
    waveMeanDirection: 186,
    waveMeanPeriod: 2,
    minWindSpeed: 1.0683246478923132,
    maxWindSpeed: 3.8642400151240097,
    avgWindSpeed: 2.4470019134494967,
    windDirection: 238,
    weeklyAlertLevel: 0,
  },
  {
    id: 1,
    date: '2022-06-25T02:59:59.999Z',
    minBottomTemperature: 0,
    maxBottomTemperature: 0,
    avgBottomTemperature: 0,
    degreeHeatingDays: 0,
    topTemperature: 0,
    satelliteTemperature: 24.00602354214886,
    minWaveHeight: 0.25399888336189147,
    maxWaveHeight: 0.7053866500812253,
    avgWaveHeight: 0.49466218581824045,
    waveMeanDirection: 184,
    waveMeanPeriod: 3,
    minWindSpeed: 2.3608638556613584,
    maxWindSpeed: 6.910529333003823,
    avgWindSpeed: 4.019357707456016,
    windDirection: 125,
    weeklyAlertLevel: 0,
  },
];
