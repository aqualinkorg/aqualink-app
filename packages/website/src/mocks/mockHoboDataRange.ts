import { TimeSeriesDataRange } from "../store/Reefs/types";

export const mockHoboDataRange: TimeSeriesDataRange = {
  hobo: {
    bottomTemperature: [
      {
        minDate: "2017-11-02T14:59:08.000Z",
        maxDate: "2020-02-02T20:59:08.000Z",
      },
    ],
    alert: [],
    dhw: [],
    satelliteTemperature: [],
    topTemperature: [],
    sstAnomaly: [],
    significantWaveHeight: [],
    wavePeakPeriod: [],
    waveMeanDirection: [],
    windSpeed: [],
    windDirection: [],
  },
  spotter: {
    bottomTemperature: [],
    alert: [],
    dhw: [],
    satelliteTemperature: [],
    topTemperature: [],
    sstAnomaly: [],
    significantWaveHeight: [],
    wavePeakPeriod: [],
    waveMeanDirection: [],
    windSpeed: [],
    windDirection: [],
  },
  sofarApi: {
    bottomTemperature: [],
    alert: [],
    dhw: [],
    satelliteTemperature: [],
    topTemperature: [],
    sstAnomaly: [],
    significantWaveHeight: [],
    wavePeakPeriod: [],
    waveMeanDirection: [],
    windSpeed: [],
    windDirection: [],
  },
};
