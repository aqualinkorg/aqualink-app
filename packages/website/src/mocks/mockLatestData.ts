import { LatestData } from 'store/Sites/types';

const aDayAgo = new Date(
  new Date().setDate(new Date().getDate() - 1),
).toISOString();

export const mockLatestData: LatestData[] = [
  {
    id: 8597,
    timestamp: aDayAgo,
    value: 0,
    source: 'noaa',
    metric: 'temp_alert',
    siteId: 1,
  },
  {
    id: 8660,
    timestamp: aDayAgo,
    value: 0,
    source: 'noaa',
    metric: 'temp_weekly_alert',
    siteId: 1,
  },
  {
    id: 8599,
    timestamp: aDayAgo,
    value: 0,
    source: 'noaa',
    metric: 'dhw',
    siteId: 1,
  },
  {
    id: 8600,
    timestamp: aDayAgo,
    value: 15.3299999237061,
    source: 'noaa',
    metric: 'satellite_temperature',
    siteId: 1,
  },
  {
    id: 3534,
    timestamp: aDayAgo,
    value: 20.160000000000004,
    source: 'spotter',
    metric: 'top_temperature',
    siteId: 1,
  },
  {
    id: 3508,
    timestamp: aDayAgo,
    value: 20.259999999999998,
    source: 'spotter',
    metric: 'bottom_temperature',
    siteId: 1,
  },
  {
    id: 63633,
    timestamp: aDayAgo,
    value: 30.177,
    source: 'sonde',
    metric: 'bottom_temperature',
    siteId: 1,
  },
  {
    id: 8598,
    timestamp: aDayAgo,
    value: -0.4800000762939014,
    source: 'noaa',
    metric: 'sst_anomaly',
    siteId: 1,
  },
  {
    id: 3450,
    timestamp: aDayAgo,
    value: 0.63,
    source: 'spotter',
    metric: 'significant_wave_height',
    siteId: 1,
  },
  {
    id: 3595,
    timestamp: aDayAgo,
    value: 5.9,
    source: 'spotter',
    metric: 'wave_mean_period',
    siteId: 1,
  },
  {
    id: 3433,
    timestamp: aDayAgo,
    value: 152.354,
    source: 'spotter',
    metric: 'wave_mean_direction',
    siteId: 1,
  },
  {
    id: 3601,
    timestamp: aDayAgo,
    value: 1.2,
    source: 'spotter',
    metric: 'wind_speed',
    siteId: 1,
  },
  {
    id: 3551,
    timestamp: aDayAgo,
    value: 262,
    source: 'spotter',
    metric: 'wind_direction',
    siteId: 1,
  },
  {
    id: 63608,
    timestamp: aDayAgo,
    value: 0.08,
    source: 'sonde',
    metric: 'cholorophyll_rfu',
    siteId: 1,
  },
  {
    id: 63610,
    timestamp: aDayAgo,
    value: 0.35,
    source: 'sonde',
    metric: 'cholorophyll_concentration',
    siteId: 1,
  },
  {
    id: 63611,
    timestamp: aDayAgo,
    value: 60847.4,
    source: 'sonde',
    metric: 'conductivity',
    siteId: 1,
  },
  {
    id: 63613,
    timestamp: aDayAgo,
    value: 7.922,
    source: 'sonde',
    metric: 'water_depth',
    siteId: 1,
  },
  {
    id: 63615,
    timestamp: aDayAgo,
    value: 111.9,
    source: 'sonde',
    metric: 'odo_saturation',
    siteId: 1,
  },
  {
    id: 63617,
    timestamp: aDayAgo,
    value: 6.9,
    source: 'sonde',
    metric: 'odo_concentration',
    siteId: 1,
  },
  {
    id: 63619,
    timestamp: aDayAgo,
    value: 36.56,
    source: 'sonde',
    metric: 'salinity',
    siteId: 1,
  },
  {
    id: 63620,
    timestamp: aDayAgo,
    value: 55372.3,
    source: 'sonde',
    metric: 'specific_conductance',
    siteId: 1,
  },
  {
    id: 63622,
    timestamp: aDayAgo,
    value: 35992,
    source: 'sonde',
    metric: 'tds',
    siteId: 1,
  },
  {
    id: 63624,
    timestamp: aDayAgo,
    value: 2.44,
    source: 'sonde',
    metric: 'turbidity',
    siteId: 1,
  },
  {
    id: 63626,
    timestamp: aDayAgo,
    value: 0,
    source: 'sonde',
    metric: 'total_suspended_solids',
    siteId: 1,
  },
  {
    id: 63627,
    timestamp: aDayAgo,
    value: 1.187,
    source: 'sonde',
    metric: 'sonde_wiper_position',
    siteId: 1,
  },
  {
    id: 63629,
    timestamp: aDayAgo,
    value: 8.26,
    source: 'sonde',
    metric: 'ph',
    siteId: 1,
  },
  {
    id: 63631,
    timestamp: aDayAgo,
    value: -80.7,
    source: 'sonde',
    metric: 'ph_mv',
    siteId: 1,
  },
  {
    id: 63635,
    timestamp: aDayAgo,
    value: 5.97,
    source: 'sonde',
    metric: 'sonde_battery_voltage',
    siteId: 1,
  },
  {
    id: 63636,
    timestamp: aDayAgo,
    value: 0,
    source: 'sonde',
    metric: 'sonde_cable_power_voltage',
    siteId: 1,
  },
];
