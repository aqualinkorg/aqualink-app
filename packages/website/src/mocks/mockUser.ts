import { User } from "../store/User/types";

export const mockUser: User = {
  adminLevel: "default",
  administeredReefs: [],
  email: "mail@mail.com",
  firebaseUid: "RaNdOm StRiNg",
  fullName: "Mock User",
  organization: "Some organization",
  token: "RaNdOm StRiNg",
};
