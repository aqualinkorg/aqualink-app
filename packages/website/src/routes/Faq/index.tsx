import React from "react";
import {
  withStyles,
  WithStyles,
  createStyles,
  Typography,
} from "@material-ui/core";

import NavBar from "../../common/NavBar";
import Footer from "../../common/Footer";

const Faq = ({ classes }: FaqProps) => {
  return (
    <>
      <NavBar searchLocation={false} />
      <div>
        {/* Main container */}
        <div className="page-container">
          {/* bloc-11 */}
          <div className="bloc l-bloc" id="bloc-11">
            <div className="container bloc-lg">
              <div className="row">
                <div className="col">
                  <Typography className={classes.title} variant="h4">
                    Frequently Asked Questions
                  </Typography>
                </div>
              </div>
              <div className="row faqq">
                <div className="col">
                  <Typography className={classes.question} variant="h5">
                    I am managing a large area and would like more than one
                    spotter, can I get multiple?
                  </Typography>
                  <p>
                    Please submit an application for each reef or location you
                    would like a spotter for and weâ€™ll be in touch to figure out
                    a staged roll out with you.
                  </p>
                </div>
              </div>
              <div className="row faqq">
                <div className="col">
                  <Typography className={classes.question} variant="h5">
                    Who is responsible for the satellite communication costs?
                  </Typography>
                  <p>
                    Aqualink will be responsible for any satellite communication
                    costs.
                  </p>
                </div>
              </div>
              <div className="row faqq">
                <div className="col">
                  <Typography className={classes.question} variant="h5">
                    What is the maximum depth a temperature sensor on the smart
                    mooring can be deployed?
                  </Typography>
                  <p>100 meters.</p>
                </div>
              </div>
              <div className="row faqq">
                <div className="col">
                  <Typography className={classes.question} variant="h5">
                    What are the shipping costs?
                    <br />
                  </Typography>
                  <p>
                    Shipping cost will vary greatly depending on your location.
                    To get an idea of shipping costs you can visit
                    https://onlineshippingcalculator.com/ and enter the
                    following:
                    <br />
                  </p>
                  <ul>
                    <li>
                      <p>Set ship from location to San Francisco CA USA</p>
                    </li>
                    <li>
                      <p>Set ship to location to your location</p>
                    </li>
                    <li>
                      <p>Weight will be 22lbs</p>
                    </li>
                    <li>
                      <p>
                        Dimension will be 18 inches x 18 inches x 14 inches.
                      </p>
                    </li>
                    <li>
                      <p>
                        Because our shipment will arrive in two boxes and using
                        our negotiated rates you will need to multiply this
                        result by 1.5x.&nbsp;
                        <br />
                      </p>
                    </li>
                    <li>
                      <p>
                        This should be a rough estimate.
                        <br />
                      </p>
                    </li>
                    <li>
                      <p>
                        This does not include total landed costs for
                        international shipping (see below).
                        <br />
                      </p>
                    </li>
                  </ul>
                </div>
              </div>
              <div className="row faqq">
                <div className="col">
                  <Typography className={classes.question} variant="h5">
                    Am I responsible for customs / duties / taxes?
                  </Typography>
                  <p>
                    Yes. As the recipient of the equipment you will need to
                    determine if you need to pay these to receive your equipment
                    at customs.
                  </p>
                </div>
              </div>
              <div className="row faqq">
                <div className="col">
                  <Typography className={classes.question} variant="h5">
                    How big is the smart mooring and spotter buoy?
                  </Typography>
                  <p>
                    The total smart mooring system is small enough for regular
                    shipping such as UPS, DHL, FedEx, etc. The dimensions and
                    weight of the buoy are:
                    <br />
                    <br />
                    42cm diameter
                    <br />
                    31cm height
                    <br />
                    5.4 KG
                    <br />
                    <br />
                    There is a ballast chain and smart mooring cable, the weight
                    and dimensions are still TBD but depending on the length of
                    cable needed for your target depth this should be around the
                    same dimensions and less than 15kg.
                    <br />
                  </p>
                </div>
              </div>
              <div className="row faqq">
                <div className="col">
                  <Typography className={classes.question} variant="h5">
                    Can I see more specification for the spotter buoy?
                  </Typography>
                  <p>
                    You can see some of the specifications&nbsp;
                    <a
                      href="https://content.sofarocean.com/hubfs/Spotter%20product%20documentation%20page/Sofar_Spotter_Specs_8_30_19.pdf"
                      target="_blank"
                      rel="noopener noreferrer"
                    >
                      here
                    </a>
                    :<br />
                    More specifications will be released soon.
                    <br />
                  </p>
                </div>
              </div>
              <div className="row faqq">
                <div className="col">
                  <Typography className={classes.question} variant="h5">
                    How is the smart mooring moored? Does it come with a mooring
                    anchor?
                  </Typography>
                  <p>
                    The smart mooring does not come with an anchor. It
                    terminates in a stainless steel shackle with a chain. This
                    can be attached to an existing mooring. We will work with
                    you to determine the best arrangement for mooring at your
                    site as factors like depth, wind, current, wave action, and
                    storms may determine the configuration of the mooring.
                    <br />
                  </p>
                </div>
              </div>
              <div className="row faqq">
                <div className="col">
                  <Typography className={classes.question} variant="h5">
                    Am I responsible for permitting and permissions to install?
                  </Typography>
                  <p>
                    Yes. You will need to know the regulations for your
                    installation site and ensure you have the proper permits and
                    permissions.
                    <br />
                  </p>
                </div>
              </div>
              <div className="row faqq">
                <div className="col">
                  <Typography className={classes.question} variant="h5">
                    How do I receive data from the smart mooring?
                  </Typography>
                  <p>
                    Data will be available in an open-source dashboard provided
                    by aqualink.org.
                    <br />
                  </p>
                </div>
              </div>
              <div className="row faqq">
                <div className="col">
                  <Typography className={classes.question} variant="h5">
                    How often will I receive data?
                  </Typography>
                  <p>
                    Data will be real time transmitted every 30-45 minutes.
                    <br />
                  </p>
                </div>
              </div>
              <div className="row faqq">
                <div className="col">
                  <Typography className={classes.question} variant="h5">
                    What data will I receive from the smart mooring?
                  </Typography>
                  <p>
                    The planned data available will be:
                    <br />
                    <br />
                    significant wave height
                    <br />
                    peak period
                    <br />
                    mean period
                    <br />
                    peak direction
                    <br />
                    mean direction
                    <br />
                    peak directional spread
                    <br />
                    mean directional spread
                    <br />
                    wind speed
                    <br />
                    wind direction
                    <br />
                    GPS coordinates
                    <br />
                    surface temperature (placement configurable)
                    <br />
                    subsurface temperature (placement configurable)
                    <br />
                  </p>
                </div>
              </div>
            </div>
          </div>
          {/* bloc-11 END */}
        </div>
      </div>
      <Footer />
    </>
  );
};

const styles = () =>
  createStyles({
    title: {
      marginTop: "1rem",
      marginBottom: "1rem",
    },
    question: {
      marginTop: "1rem",
      marginBottom: "1rem",
      fontWeight: "bold",
    },
    root: {
      top: 10,
      height: "100%",
    },
    map: {
      height: "100%",
    },
    reefTable: {
      height: "calc(100vh - 64px)",
      overflowY: "auto",
    },
  });

type FaqProps = WithStyles<typeof styles>;

export default withStyles(styles)(Faq);
