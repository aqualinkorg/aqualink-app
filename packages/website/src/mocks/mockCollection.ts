import { CollectionDetails } from "../store/Collection/types";
import { mockReef } from "./mockReef";
import { mockUser } from "./mockUser";

export const mockCollection: CollectionDetails = {
  id: 1,
  isPublic: false,
  name: "Mock Collection",
  user: mockUser,
  reefs: [mockReef],
  reefIds: [1],
};
