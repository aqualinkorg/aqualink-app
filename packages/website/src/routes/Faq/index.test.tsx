import React from "react";
import { Provider } from "react-redux";
import { render } from "@testing-library/react";
import configureStore from "redux-mock-store";
import { BrowserRouter as Router } from "react-router-dom";
import { mockUser } from "../../mocks/mockUser";

import Faq from ".";

jest.mock("../../common/NavBar", () => "Mock-NavBar");
jest.mock("../../common/Footer", () => "Mock-Footer");

const mockStore = configureStore([]);
describe("Faq", () => {
  let element: HTMLElement;
  beforeEach(() => {
    const store = mockStore({
      user: {
        userInfo: mockUser,
        loading: false,
        error: null,
      },
    });

    store.dispatch = jest.fn();

    element = render(
      <Provider store={store}>
        <Router>
          <Faq />
        </Router>
      </Provider>
    ).container;
  });

  it("should render with given state from Redux store", () => {
    expect(element).toMatchSnapshot();
  });
});
