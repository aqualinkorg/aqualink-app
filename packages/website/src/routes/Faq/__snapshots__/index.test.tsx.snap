// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Faq should render with given state from Redux store 1`] = `
<div>
  <mock-navbar
    searchlocation="false"
  />
  <div>
    <div
      class="page-container"
    >
      <div
        class="bloc l-bloc"
        id="bloc-11"
      >
        <div
          class="container bloc-lg"
        >
          <div
            class="row"
          >
            <div
              class="col"
            >
              <mock-typography
                classname="Faq-title-1"
                variant="h4"
              >
                Frequently Asked Questions
              </mock-typography>
            </div>
          </div>
          <div
            class="row faqq"
          >
            <div
              class="col"
            >
              <mock-typography
                classname="Faq-question-2"
                variant="h5"
              >
                I am managing a large area and would like more than one spotter, can I get multiple?
              </mock-typography>
              <p>
                Please submit an application for each reef or location you would like a spotter for and we’ll be in touch to figure out a staged roll out with you.
              </p>
            </div>
          </div>
          <div
            class="row faqq"
          >
            <div
              class="col"
            >
              <mock-typography
                classname="Faq-question-2"
                variant="h5"
              >
                Who is responsible for the satellite communication costs?
              </mock-typography>
              <p>
                Aqualink will be responsible for any satellite communication costs.
              </p>
            </div>
          </div>
          <div
            class="row faqq"
          >
            <div
              class="col"
            >
              <mock-typography
                classname="Faq-question-2"
                variant="h5"
              >
                What is the maximum depth a temperature sensor on the smart mooring can be deployed?
              </mock-typography>
              <p>
                100 meters.
              </p>
            </div>
          </div>
          <div
            class="row faqq"
          >
            <div
              class="col"
            >
              <mock-typography
                classname="Faq-question-2"
                variant="h5"
              >
                What are the shipping costs?
                <br />
              </mock-typography>
              <p>
                Shipping cost will vary greatly depending on your location. To get an idea of shipping costs you can visit https://onlineshippingcalculator.com/ and enter the following:
                <br />
              </p>
              <ul>
                <li>
                  <p>
                    Set ship from location to San Francisco CA USA
                  </p>
                </li>
                <li>
                  <p>
                    Set ship to location to your location
                  </p>
                </li>
                <li>
                  <p>
                    Weight will be 22lbs
                  </p>
                </li>
                <li>
                  <p>
                    Dimension will be 18 inches x 18 inches x 14 inches.
                  </p>
                </li>
                <li>
                  <p>
                    Because our shipment will arrive in two boxes and using our negotiated rates you will need to multiply this result by 1.5x. 
                    <br />
                  </p>
                </li>
                <li>
                  <p>
                    This should be a rough estimate.
                    <br />
                  </p>
                </li>
                <li>
                  <p>
                    This does not include total landed costs for international shipping (see below).
                    <br />
                  </p>
                </li>
              </ul>
            </div>
          </div>
          <div
            class="row faqq"
          >
            <div
              class="col"
            >
              <mock-typography
                classname="Faq-question-2"
                variant="h5"
              >
                Am I responsible for customs / duties / taxes?
              </mock-typography>
              <p>
                Yes. As the recipient of the equipment you will need to determine if you need to pay these to receive your equipment at customs.
              </p>
            </div>
          </div>
          <div
            class="row faqq"
          >
            <div
              class="col"
            >
              <mock-typography
                classname="Faq-question-2"
                variant="h5"
              >
                How big is the smart mooring and spotter buoy?
              </mock-typography>
              <p>
                The total smart mooring system is small enough for regular shipping such as UPS, DHL, FedEx, etc. The dimensions and weight of the buoy are:
                <br />
                <br />
                42cm diameter
                <br />
                31cm height
                <br />
                5.4 KG
                <br />
                <br />
                There is a ballast chain and smart mooring cable, the weight and dimensions are still TBD but depending on the length of cable needed for your target depth this should be around the same dimensions and less than 15kg.
                <br />
              </p>
            </div>
          </div>
          <div
            class="row faqq"
          >
            <div
              class="col"
            >
              <mock-typography
                classname="Faq-question-2"
                variant="h5"
              >
                Can I see more specification for the spotter buoy?
              </mock-typography>
              <p>
                You can see some of the specifications 
                <a
                  href="https://content.sofarocean.com/hubfs/Spotter%20product%20documentation%20page/Sofar_Spotter_Specs_8_30_19.pdf"
                  rel="noopener noreferrer"
                  target="_blank"
                >
                  here
                </a>
                :
                <br />
                More specifications will be released soon.
                <br />
              </p>
            </div>
          </div>
          <div
            class="row faqq"
          >
            <div
              class="col"
            >
              <mock-typography
                classname="Faq-question-2"
                variant="h5"
              >
                How is the smart mooring moored? Does it come with a mooring anchor?
              </mock-typography>
              <p>
                The smart mooring does not come with an anchor. It terminates in a stainless steel shackle with a chain. This can be attached to an existing mooring. We will work with you to determine the best arrangement for mooring at your site as factors like depth, wind, current, wave action, and storms may determine the configuration of the mooring.
                <br />
              </p>
            </div>
          </div>
          <div
            class="row faqq"
          >
            <div
              class="col"
            >
              <mock-typography
                classname="Faq-question-2"
                variant="h5"
              >
                Am I responsible for permitting and permissions to install?
              </mock-typography>
              <p>
                Yes. You will need to know the regulations for your installation site and ensure you have the proper permits and permissions.
                <br />
              </p>
            </div>
          </div>
          <div
            class="row faqq"
          >
            <div
              class="col"
            >
              <mock-typography
                classname="Faq-question-2"
                variant="h5"
              >
                How do I receive data from the smart mooring?
              </mock-typography>
              <p>
                Data will be available in an open-source dashboard provided by aqualink.org.
                <br />
              </p>
            </div>
          </div>
          <div
            class="row faqq"
          >
            <div
              class="col"
            >
              <mock-typography
                classname="Faq-question-2"
                variant="h5"
              >
                How often will I receive data?
              </mock-typography>
              <p>
                Data will be real time transmitted every 30-45 minutes.
                <br />
              </p>
            </div>
          </div>
          <div
            class="row faqq"
          >
            <div
              class="col"
            >
              <mock-typography
                classname="Faq-question-2"
                variant="h5"
              >
                What data will I receive from the smart mooring?
              </mock-typography>
              <p>
                The planned data available will be:
                <br />
                <br />
                significant wave height
                <br />
                peak period
                <br />
                mean period
                <br />
                peak direction
                <br />
                mean direction
                <br />
                peak directional spread
                <br />
                mean directional spread
                <br />
                wind speed
                <br />
                wind direction
                <br />
                GPS coordinates
                <br />
                surface temperature (placement configurable)
                <br />
                subsurface temperature (placement configurable)
                <br />
              </p>
            </div>
          </div>
          <div
            class="row faqq"
          >
            <div
              class="col"
            >
              <mock-typography
                classname="Faq-question-2"
                variant="h5"
              >
                What are the best practices for conducting a survey?
              </mock-typography>
              <p>
                We have created an animated field guide that will describe the best practices for conducting a survey and using the Aqualink system.
                <br />
                <br />
                <div
                  style="width: 100%; height: 360px;"
                  title="field_guide"
                />
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <mock-footer />
</div>
`;
